src/hotspot/share/runtime/arguments.cpp

```diff
@@ -3152,6 +3152,17 @@ jint Arguments::finalize_vm_init_args(bool patch_mod_javabase) {
 #ifdef ZERO
   // Zero always runs in interpreted mode
   set_mode_flags(_int);
+
+  // Enable error context decoding on known platforms
+#if defined(IA32) || defined(AMD64) || defined(ARM) || \
```

Comment:
What's the reasoning behind putting it here vs `src/hotspot/cpu/zero/vm_version_zero.cpp`?

Reply:
No good reason, really. I moved this to `vm_version_zero.cpp`.